inc/class-command.php

```diff
@@ -151,4 +151,16 @@ public function jobs( $args, $assoc_args  ) {
 		}
 
 	}
+
+	/**
+	 * Upgrade to the latest database schema.
+	 */
+	public function upgrade( ) {
+		if ( Upgrade\upgrade_database() ) {
```

Comment:
Can this be run automatically, or are we thinking we'll need to run this on every site manually?

Reply:
I was thinking to run in manually on every site, in part to avoid any performance issues around running the `ALTER` query on large tables. I'm not sure if this is a real or imagined issue.

As it currently stands, there is a bug around rescheduling that I need to figure out. Depending on the approach, I may need to modify each row as part of the upgrade routine.

Reply:
After some testing this morning, I did need to add a data upgrade routine in bc505402264d021627581f44ccfb201e044a251f